rero_ils/modules/items/api/record.py

```diff
@@ -510,3 +510,17 @@ def is_new_acquisition(self):
             return datetime.strptime(
                 acquisition_date, '%Y-%m-%d') < datetime.now()
         return False
+
+    @classmethod
+    def get_number_masked_items_by_holdings_pid(cls, holding_pid):
+        """Returns the number of unmasked items and attached to a holding.
+
+        :param holding_pid: the pid of the holdings.
+        :return number of un masked items.
+        """
+        from . import ItemsSearch
```

Comment:
```from api import ItemsSearch``` ? Don't understand why `from .` is working :s

Reply:
because the `ItemsSearch` is in `..modules.items.api.__init__.py`